internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -804,6 +794,8 @@ func (t *Translator) buildGlobalRateLimit(policy *egv1a1.BackendTrafficPolicy) (
 		if err != nil {
 			return nil, err
 		}
+		// Set the Name field as <policy-ns>/<policy-name>/rule/<rule-index>
+		irRules[i].Name = fmt.Sprintf("%s/%s/rule/%d", policy.Namespace, policy.Name, i)
```

Comment:
non blovking nit: helper func https://github.com/envoyproxy/gateway/blob/0752df11ec2a145e424d6d926a851c426b6c07ba/internal/gatewayapi/helpers.go#L444 for this